admin/class-yoast-notifications.php

```diff
@@ -202,36 +202,21 @@ private function get_notification_from_ajax_request() {
 	}
 
 	/**
-	 * Show the alerts overview page.
+	 * Collect the notifications and group them together.
 	 */
-	public static function show_overview_page() {
-
-		/**
-		 * Stops PHPStorm from nagging about this variable being unused. The variable is used in the view.
-		 *
-		 * @noinspection PhpUnusedLocalVariableInspection
-		 */
-		$alerts_data = self::get_template_variables();
-
-		include WPSEO_PATH . 'admin/views/alerts-dashboard.php';
-	}
```

Comment:
This is the unused function, is that correct?

Reply:
Yes